### ISSUE TITLE
Does horovod support sync BN?

### ISSUE THREAD

**Issue by MingYang-buaa** [TRI-ML/packnet-sfm#71]
Hi, I'm newer to horovod. And I'm trying to retrain SelfSupModel with Resnet18. So does horovod support sync BN which is important for distributed training?

**Comment by VitorGuizilini-TRI** [comment]
As far as I know yes, we have used it successfully in the past.

**Comment by MingYang-buaa** [comment]
Thanks for your quick reply, I find it and replaced nn.BatchNorm2d with hvd.SyncBatchNorm. But the GPU-Util down from 90% to 30%. Is't normal? 
Is there another way?

**Comment by VitorGuizilini-TRI** [comment]
That doesn't sound normal, did you try training? What sort of results are you getting?

**Comment by MingYang-buaa** [comment]
Sorry for the late reply.
Yeah, I trained it with SelfSupModel (DepthResNet and PoseResNet), just replaced nn.BatchNorm2d with hvd.SyncBatchNorm. And I got the abs_rel as 0.121(origin version is 0.118). That's strange.

**Comment by VitorGuizilini-TRI** [comment]
I think that's within the normal variation we see in multiple training sessions. The significant decrease in GPU-Util is a little bit concerning though.
